config/v1/config.go

```diff
@@ -10,63 +10,64 @@ import (
 	"github.com/chanzuckerberg/fogg/plugins"
 	"github.com/hashicorp/go-multierror"
 	"gopkg.in/go-playground/validator.v9"
+	"gopkg.in/yaml.v2"
 )
 
 type TfLint struct {
-	Enabled *bool `json:"enabled,omitempty"`
+	Enabled *bool `json:"enabled,omitempty" yaml:"enabled,omitempty"`
 }
 
 type Defaults struct {
-	AccountID          int64             `json:"account_id,omitempty" validate:"required"`
-	AWSProfileBackend  string            `json:"aws_profile_backend" validate:"required"`
-	AWSProfileProvider string            `json:"aws_profile_provider" validate:"required"`
-	AWSProviderVersion string            `json:"aws_provider_version" validate:"required"`
-	AWSRegionBackend   string            `json:"aws_region_backend" validate:"required"`
-	AWSRegionProvider  string            `json:"aws_region_provider" validate:"required"`
-	AWSRegions         []string          `json:"aws_regions,omitempty"`
-	ExtraVars          map[string]string `json:"extra_vars"`
-	InfraBucket        string            `json:"infra_s3_bucket" validate:"required"`
-	InfraDynamoTable   string            `json:"infra_dynamo_db_table"`
-	Owner              string            `json:"owner" validate:"required"`
-	Project            string            `json:"project" validate:"required"`
-	TerraformVersion   string            `json:"terraform_version" validate:"required"`
-	TfLint             *TfLint           `json:"tflint,omitempty"`
+	AccountID          int64             `json:"account_id,omitempty" yaml:"account_id,omitempty" validate:"required"`
+	AWSProfileBackend  string            `json:"aws_profile_backend" yaml:"aws_profile_backend" validate:"required"`
```

Comment:
No need to support yaml for v1

Reply:
If I remove yaml support for v1 I fail tests in plan_test.go (Something to do with the resolvers)